index.html

```diff
@@ -315,6 +315,22 @@ <h2 class="testimonial__title">Client Testimonial</h2>
                 </div>
             </div>
         </section>
+
+        <!-- Contacts section -->
+        <section class="contacts">
+            <div class="contacts__box">
+                <span class="contacts__label">Get in touch</span>
+                <h2 class="contacts__title">Just Say Hello!</h2>
+                <button class="contacts__button">Contact</button>
+                <div class="contacts__links">
+                    <a class="contacts__link" href="https://facebook.com" target="_blank" rel="noreferrer"><img class="contacts__icon" src="./images/Contacts/facebook.svg" alt="facebook icon" /></a>
+                    <a class="contacts__link" href="https://linkedin.com" target="_blank" rel="noreferrer"><img class="contacts__icon" src="./images/Contacts/linkedin-logo.svg" alt="linkedin icon" /></a>
+                    <a class="contacts__link" href="https://behance.com" target="_blank" rel="noreferrer"><img class="contacts__icon" src="./images/Contacts/behance.svg" alt="behance icon" /></a>
+                    <a class="contacts__link" href="https://twitter.com" target="_blank" rel="noreferrer"><img class="contacts__icon" src="./images/Contacts/twitter.svg" alt="twitter icon" /></a>
+                    <a class="contacts__link" href="https://dribbble.com" target="_blank" rel="noreferrer"><img class="contacts__icon" src="./images/Contacts/dribbble.svg" alt="dribbble icon" /></a>
```

Comment:
On mobile you don't change the size of the svg, but instead the size of the `a` elements. This results in the svgs being the wrong size.
![image](https://user-images.githubusercontent.com/68795596/208673424-c91883ae-2354-4f40-8eb8-bbcae6d3aa05.png)